test/test_language.rb

```diff
@@ -343,6 +345,8 @@ def test_extensions
     assert Language['LSL'].extensions.include?('.lsl')
     assert Language['Perl'].extensions.include?('.pl')
     assert Language['Python'].extensions.include?('.py')
+    assert Language['Hack'].extensions.include?('.hack')
+    assert Language['Hack'].extensions.include?('.hhi')
```

Comment:
You can remove these tests. We already have tests in place to check this. I'm not even sure what these hardcoded tests are used for.

Reply:
:+1: